tests/integration_test.rs

```diff
@@ -268,3 +274,82 @@ async fn test_pypi_ignore(
     let pack_result = pixi_pack::pack(pack_options).await;
     assert_eq!(pack_result.is_err(), should_fail);
 }
+#[rstest]
+#[tokio::test]
+async fn test_run_packed_executable(options: Options, required_fs_objects: Vec<&'static str>) {
```

Comment:
i'm wondering whether we can automatically test cross-platform packs in ci 🤔 although i can't think of a pretty way to do so... we should still at least test once that it works

Reply:
That would be amazing.

Reply:
this is not necessary anymore since we ensure the shasum is reproducible between platforms